### DISCUSSION TITLE
Multi-speaker TTS training

### DISCUSSION THREAD

**Discussion by oytunturk** [NVIDIA-NeMo/NeMo#3448] [Q&A] [answered]
Is there a Nemo recipe that supports multi-speaker TTS training using speaker embeddings?

**Comment by XuesongYang** [comment]
We are working on the feature. @subhankar-ghosh, could you please share guidance here?

**Comment by subhankar-ghosh** [comment, marked as answer]
Yes, we are working on this feature where we want to generate speaker embeddings using Speaker Verification model and use them to condition FastPitch. It is still Work in Progress. Thanks for the question.
